### DISCUSSION TITLE
Should messages be repeated when a test fails?

### DISCUSSION THREAD

**Discussion by forrestab** [thomhurst/TUnit#1266] [Q&A] [answered]
Using TUnit@0.3.34, when a test fails the message is repeated several times in VS2022 17.12.1 Pro and the message + stacktrace is repeated running from the dotnet cli.

Sample test

```c#
public sealed class SampleTests
{
    [Test]
    public async Task simple_input_is_added_to_existing_json_and_outputs_valid_json()
    {
        string empty = "";
        await Assert.That(empty).IsNotEmpty();
    }
}
```

From dotnet cli
![image](https://github.com/user-attachments/assets/588d5b00-323c-4e94-90d4-4a1fdb21844b)

From VS2022
![image](https://github.com/user-attachments/assets/6af3dc2d-8234-40c9-99dd-9f896fd09cf9)

**Comment by thomhurst** [comment]
Yeah this should be fixed in the next update, hopefully tomorrow

**Comment by thomhurst** [comment, marked as answer]
Should be fixed in 0.3.43

**Reply by forrestab** [reply]
Thank you, the output is what is expected now.
